### PR TITLE
Get expiry from session token

### PR DESCRIPTION
**Pull request by snvijaya**
Get expiry from session token

### CODE REVIEW

File: hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/HttpHeaderConfigurations.java [hunk 1]
```diff
@@ -70,8 +70,7 @@
   public static final String X_MS_LEASE_ID = "x-ms-lease-id";
   public static final String X_MS_PROPOSED_LEASE_ID = "x-ms-proposed-lease-id";
   public static final String X_MS_LEASE_BREAK_PERIOD = "x-ms-lease-break-period";
-  public static final String X_MS_FASTPATH_SESSION_AUTH = "x-ms-fastpath-session-auth";
-  public static final String X_MS_FASTPATH_SESSION_DATA = "x-ms-fastpath-session-data";
+  public static final String X_MS_FASTPATH_SESSION_EXPIRY = "x-ms-fastpath-session-expiry";
```
Comment (vinaysbadami) [inline hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/HttpHeaderConfigurations.java]: expiry-time

File: hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsFastpathSession.java [hunk 2]
```diff
@@ -201,8 +208,8 @@ boolean fetchFastpathSessionToken() {
       AbfsRestOperation op = executeFetchFastpathSessionToken();
```
Comment (vinaysbadami) [inline hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsFastpathSession.java:208]: is it gauranteed at line 209 that the op has succeeded? u did not get a unauthorized, server error ec?

Comment (snvijaya) [reply hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsFastpathSession.java:208]: The line above will throw an exception which is caught and will move the connection to REST.

File: hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/HttpHeaderConfigurations.java (hunk 1 above)
Comment (snvijaya) [reply hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/HttpHeaderConfigurations.java]: Done

**Pull request merged**